src/tests/unit/common/__snapshots__/element-snapshot-formatter.test.ts.snap

```diff
@@ -0,0 +1,25 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`element-snapshot-formatter format html for snapshot normalizing ms-Panel-headerText header-text--0nPhV headerText-198 1`] = `
+<DocumentFragment>
+  <div>
+    <div
+      class="ms-Panel-headerText header-text--{{CSS_MODULE_HASH}} headerText-198"
```

Comment:
Shouldn't this be `headerText-000`?

Reply:
(I imagine this from the module normalization applying first and then the `{` and `}` characters confusing the OF normalization regex)

Reply:
This is fixed now.